MapIntegerHW/src/Main.java

```diff
@@ -0,0 +1,27 @@
+import java.util.HashMap;
+import java.util.Map;
+
+public class Main {
+
+	public static void main(String[] args) {
+		
+		System.out.println(getWordCount("Ivan"));
+
+	}
+	public static String[] voc = {"Ivan", "Anna","Ivan","Ivan","Anna","Petr","lena","lena","lena","lena"};
+	public static Map<String,Integer> map = new HashMap<>();
+	
+	public static Map<String, Integer> getWordCount(String strings)  {
+		for (String s : voc ) {
+			if(!map.containsKey(s)) {
+				map.put(s,1);
+				
+			}else {
+				int count = map.get(s);
+				map.put(s, count+1);
+			}
+		}
+		return map;
```

Comment:
We have another notation to express `if` statement `if(a){t1} else {t2}` -> `a?t1:t2`

Think how to rewrite the code in 2 lines instead of long if-then statement.